echo-pubsub-core/src/main/java/com/netflix/spinnaker/echo/pubsub/PubsubMessageHandler.java

```diff
@@ -141,4 +166,10 @@ private void processEvent(MessageDescription description) {
     event.setDetails(details);
     pubsubEventMonitor.processEvent(event);
   }
+
+  Id getDuplicateMetricId(MessageDescription messageDescription) {
```

Comment:
I'm a fan of this, would like to see more metrics for various other paths here, like number of messages nacked, etc. Would give us confidence that we're not being inefficient anywhere.

Reply:
Ah, I have those metrics for the amazon message handlers, but I'll bump some up to this class.